datashape/coretypes.py

```diff
@@ -814,12 +834,33 @@ def restype(self):
     def argtypes(self):
         return self.parameters[:-1]
 
-    # def __repr__(self):
-    #     return " -> ".join(map(repr, self.parameters))
-
     def __str__(self):
-        return '(%s) -> %s' % (', '.join(map(str, self.parameters[:-1])),
-                               self.parameters[-1])
+        return '(%s) %s %s' % (', '.join(map(str, self.argtypes)),
+                               self.maps_to_char,
+                               self.restype)
+
+
+class ForeignKey(Function):
+    """Type for describing foreign key relationships
+    """
+    maps_to_char = '>>'
```

Comment:
do you think this could be `=>` or something? the `>>` is overloaded with bitshifting and stream stuff